src-tauri/src/manager.rs

```diff
@@ -90,12 +98,12 @@ fn handle(rx: Receiver<WatcherMessage>, state: Arc<Mutex<ManagerState>>) {
     }
 }
 
-fn start_watcher(name: &'static str, tx: Sender<WatcherMessage>) {
+fn start_watcher(name: &'static str, state: Arc<Mutex<ManagerState>>, tx: Sender<WatcherMessage>) {
     thread::spawn(move || {
         // Start the child process
         let path = name;
         let args = ["--testing", "--port", "5699"];
-        let child = Command::new(path)
+        let mut child = Command::new(path)
```

Comment:
Why does this need to be `mut`?

Reply:
This was me fighting with the borrow checker. It wouldn't work without mut